COMETwebapp/Components/Viewer/BabylonCanvasBase.cs

```diff
@@ -122,31 +113,47 @@ protected async override Task OnAfterRenderAsync(bool firstRender)
         /// </summary>
         private void InitializeElements()
         {
-            if(this.SessionAnchor != null)
+            var iteration = this.SessionAnchor?.OpenIteration;
+
+            var elementUsages = this.SessionAnchor?.OpenIteration?.Element.SelectMany(ed => ed.ContainedElement).OrderBy(x=>x.Name).ToList();
```

Comment:
I think in the UI you need to implement for the user to select the Option and ignore the EUs that are excluded in that Option.

Reply:
I think that is another issue. [https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/issues/166](url)